client/styles/components/_toolbar.scss

```diff
@@ -63,7 +63,10 @@
 }
 
 .toolbar {
-	padding: #{20 / $base-font-size}rem #{34 / $base-font-size}rem;
+	padding: #{20 / $base-font-size}rem #{15 / $base-font-size}rem;
```

Comment:
also the changes here, i feel like they're helping a little but overall the toolbar still doesn't look great. i'd prefer to hide some of these elements and be able to drill down into them.

Reply:
I think I should revert these changes for now and devote this pr to the form responsive commits only. After a solution to navbar and toolbar appearance, I can create a seperate pr.